src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj.filters

```diff
@@ -0,0 +1,159 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <Natvis Include="$(SolutionDir)tools\ConsoleTypes.natvis" />
+  </ItemGroup>
+  <ItemGroup>
+    <PRIResource Include="../Resources/en-US/Resources.resw" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="../init.cpp" />
+    <ClCompile Include="../Tab.cpp" />
+    <ClCompile Include="../Utils.cpp" />
+    <ClCompile Include="pch.cpp" />
+    <ClCompile Include="$(GeneratedFilesDir)module.g.cpp" />
+    <ClCompile Include="../AzureCloudShellGenerator.cpp">
+      <Filter>profileGeneration</Filter>
+    </ClCompile>
+    <ClCompile Include="../PowershellCoreProfileGenerator.cpp">
+      <Filter>profileGeneration</Filter>
+    </ClCompile>
+    <ClCompile Include="../WslDistroGenerator.cpp">
+      <Filter>profileGeneration</Filter>
+    </ClCompile>
+    <ClCompile Include="../AppKeyBindingsSerialization.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../CascadiaSettings.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../CascadiaSettingsSerialization.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../GlobalAppSettings.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../KeyChordSerialization.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../Profile.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../ColorScheme.cpp">
+      <Filter>settings</Filter>
+    </ClCompile>
+    <ClCompile Include="../Pane.cpp">
+      <Filter>pane</Filter>
+    </ClCompile>
+    <ClCompile Include="../DefaultProfileUtils.cpp">
+      <Filter>profileGeneration</Filter>
+    </ClCompile>
+    <ClCompile Include="$(OpenConsoleDir)\dep\jsoncpp\jsoncpp.cpp">
+      <Filter>json</Filter>
+    </ClCompile>
+    <ClCompile Include="../JsonUtils.cpp">
+      <Filter>json</Filter>
+    </ClCompile>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="../Tab.h" />

```

Comment:
Does `Tab` warrant it's own folder too?

Reply:
that's only 2 files (whereas pane's gonna get more) but it very well may